### ISSUE TITLE
[Flatpak] Have remote option

### ISSUE THREAD

**Issue by TheEvilSkeleton** [openSUSE/open-build-service#11606]
Hi,

I recently got interested into OBS. I noticed that it now supports building Flatpak bundles and I find that amazing!

However, I believe there should be a remote option too, where each Flatpak package is also its own remote, so users that want to install a Flatpak package can get updates too. I am aware that Flathub exists, but Flathub is better suited for releases, where OBS could be used for pure bleeding edge.

**Comment by hennevogel** [comment]
@perlpunk maybe something for you?

**Comment by perlpunk** [comment]
I'm not sure I understand the request.
Is it about OBS acting as a Flatpak remote server like Flathub? I think that's not trivial, and last time I looked for documentation about an API for Flathub I didn't find anything useful.

**Comment by TheEvilSkeleton** [comment]
> Is it about OBS acting as a Flatpak remote server like Flathub?

Correct.

And I don't think we need a Flathub API? Flathub uses [flat-manager](https://github.com/flatpak/flat-manager) which was made from Flatpak developers from what I know.

Here's the documentation for it: https://docs.flatpak.org/en/latest/hosting-a-repository.html

**Comment by Vogtinator** [comment]
FYI, flatpak supports OCI remotes so you can expose flatpak images on `registry.opensuse.org`.

**Comment by TheEvilSkeleton** [comment]
Yup. However, it's undocumented, so you might want to contact the Fedora developers and ask how they set up their remote, because they use the OCI format. I tried to get it working on GitLab CI[1] and GitHub Actions[2].

1. https://gitlab.com/TheEvilSkeleton/flatpak-gitlab-ci/-/blob/main/flatpak-gitlab-ci.yml
2. https://github.com/TheEvilSkeleton/flatpak-remote/blob/main/.github/workflows/flatpak.yml

**Comment by orowith2os** [comment]
Bump, since I'd like to use the OBS to distribute some of my own flatpaks from. The documentation still might not be there, but it's definitely worth a look. Would make distributing apps even easier for devs, too, specifically for nightlies.
